hopsworks-common/pom.xml

```diff
@@ -247,10 +247,12 @@
           <groupId>io.hops.hive</groupId>
           <artifactId>hive-metastore</artifactId>
         </exclusion>
+        <!--
```

Comment:
@Limmen can you remove this commented out code?

Reply:
@tkakantousis Fixed